src/plugins/data/common/search/poll_search.ts

```diff
@@ -41,8 +41,13 @@ export const pollSearch = <Response extends IKibanaSearchResponse>(
       throw new AbortError();
     }
 
+    const safeCancel = () =>
+      cancel?.().catch((e) => {
+        console.error(e); // eslint-disable-line no-console
```

Comment:
Yeah, since it's used on both the client and server this presents some challenges. I tried passing in the `Logger` client-side and I'm not sure where it actually logs to because I don't see it in the console. It's also just a bit convoluted since we have to pass the logger all the way from the plugin down to `pollSearch`.